data/mods/Magiclysm/worldgen/orc_village.json

```diff
@@ -9,6 +9,7 @@
     "method": "json",
     "object": {
       "fill_ter": "t_dirt",
+      "om_terrain": [ "orc_village" ],
```

Comment:
I suspect creating this as an overmap terrain will be needed for even debug spawning of the village.

Reply:
Yes, it was causing an error while testing. How do I create overmap terrain?

Reply:
Just in worldgen, AFAIK. It can either be in the orc_village.json file itself or in the overmap_terrain.json (and I think overmap_specials.json) file(s).

Reply:
creating special has created another syntax error.